libsvm-kitware/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+cd TPL/libsvm-3.1-custom
+
+make lib
+mkdir $PREFIX/lib
+mv libsvm* $PREFIX/lib/
```

Comment:
Going to also have to move/copy the python/svm.py and python/svmutil.py files into the site-packages directory. Optionally, we could move the svm.h heading to the ${PREFIX}/include.